packages/app/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Image, StyleSheet, Dimensions } from 'react-native';
+import { Card, Divider } from 'react-native-elements';
```

Comment:
Please create your own components. Please do not use react-native-elements or any third-party components outside.